game_init.pl

```diff
@@ -105,3 +104,20 @@
     nl,
     replay
 )).
+
+
+add_killing_achiev :-
+	not(is_alive(spider)),
+	not(is_alive(dragon)),
+	not(is_alive(snake(west))),
+	not(is_alive(snake(south))),
+	add_achievement("You have defeated every enemy that stood in your path. You truly are an ultimate slayer.").
+	
+add_killing_achiev :-
+	spider_gone,
```

Comment:
Jak dla mnie to właśnie nie. Nie wiem, jak wy, ale ja miałem taką koncepcję achievementów, że trzeba przejść grę kilka razy na różny sposób, żeby zebrać je wszystkie, dlatego niektóre osiągnięcia mogą się wykluczać w obrębie jednej rozgrywki.